code/game/objects/items.dm

```diff
@@ -676,6 +680,15 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
 
+/obj/item/proc/grind_requirements(obj/machinery/reagentgrinder/R) //Used to check for extra requirements for grinding an object
+	return TRUE
+
+ //Called BEFORE the object is ground up - use this to change grind results based on conditions
+ //Use "return -1" to prevent the grinding from occurring
+/obj/item/proc/on_grind(obj/machinery/reagentgrinder/R) //Note that it isn't necessary to include the argument if you don't mess with the reagent grinder at all

```

Comment:
Remove that last comment, we don't condone messing with base proc args without good reason

Reply:
It occurs to me, actually, that I currently don't use the argument in any instance of it, so I believe I'll snip it out unless something has need of it.